server/routes/veracode.ts

```diff
@@ -0,0 +1,86 @@
+import { type RequestHandler, Router } from 'express'
+import dayjs from 'dayjs'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+import { getFormattedName, veracodeFilters } from '../utils/utils'
+
+export default function routes({ serviceCatalogueService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  // const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+
+  get('/veracode', async (req, res) => {
```

Comment:
So, you don't need `/veracode` here and all other routes in here, the initial `/veracode` is in `server/app.ts`. So basically `app.use('/veracode', veracodeRoutes(services))` is saying prefix every route in `veracodeRoutes` with `/veracode` which means what you've added here is `/veracode/veracode` 🙂 

Basically in this file remove `/veracode` in all the `get` and `post` statements you want to use 👍

Reply:
Corrected